repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/guidance-main.ftl

```diff
@@ -5,39 +5,44 @@
 
 <#-- @ftlvariable name="document" type="uk.nhs.hee.web.beans.Guidance" -->
 <#if document??>
-    <div class="nhsuk-grid-row">
-        <div class="nhsuk-grid-column-two-thirds">
-            <h1>${document.title}</h1>
-        </div>
-    </div>
+    <main id="maincontent" role="main" class="nhsuk-main-wrapper">
+        <div class="nhsuk-width-container">
+            <div class="nhsuk-grid-row">
+                <div class="nhsuk-grid-column-two-thirds">
+                    <h1>${document.title}</h1>
+                </div>
+            </div>
 
-    <article>
-        <div class="nhsuk-grid-row">
-            <div class="nhsuk-grid-column-two-thirds">
-                <section class="nhsuk-page-content__section-one">
-                    <div class="nhsuk-page-content">
-                        <p>${document.summary}</p>
+            <article>
+                <div class="nhsuk-grid-row">
+                    <div class="nhsuk-grid-column-two-thirds">
+                        <section class="nhsuk-page-content__section-one">
+                            <div class="nhsuk-page-content">
+                                <p>${document.summary}</p>
 
-                        <#if document.contentBlocks??>
-                            <#list document.contentBlocks as block>
-                                <#if hst.isBeanType(block, 'uk.nhs.hee.web.beans.PageLastNextReview')>
-                                    <@lastNextReviewedDate lastNextReviewedDate=block/>
-                                <#elseif hst.isBeanType(block, 'org.hippoecm.hst.content.beans.standard.HippoMirror')>
-                                    <@hst.link var="img" hippobean=block/>
-                                    <img src="${img}" alt=""/>
-                                <#elseif hst.isBeanType(block, 'org.hippoecm.hst.content.beans.standard.HippoHtml')>
-                                    <@hst.html hippohtml=block/>
+                                <#if document.contentBlocks??>
+                                    <#list document.contentBlocks as block>
+                                        <#if hst.isBeanType(block, 'org.hippoecm.hst.content.beans.standard.HippoMirror')>
+                                            <@hst.link var="img" hippobean=block/>
+                                            <img src="${img}" alt=""/>
```

Comment:
Might be worth adding a ticket in the backlog to add a custom image set (https://documentation.bloomreach.com/14/library/concepts/images-and-assets/create-a-custom-image-set.html) containing `altText` field so that alt text could be content managed.

Reply:
Yes. We can cater for that on this ticket https://manifesto.atlassian.net/browse/HEE-82